backend/core/src/shared/views/partials/head.hbs

```diff
@@ -318,6 +325,10 @@
       margin-top: 3rem;
     }
 
+    .my3 {
+      margin: 3rem auto 3rem auto;
```

Comment:
real minor. this can be slightly shortened to `margin: 3rem auto` https://developer.mozilla.org/en-US/docs/Web/CSS/margin